mon-pix/tests/acceptance/recover-account-after-leaving-sco_test.js

```diff
@@ -25,9 +26,9 @@ describe('Acceptance | RecoverAccountAfterLeavingScoRoute', function() {
     });
   });
 
-  context('when account recovery is enabled', function() {
+  context('when account recovery is enabled', () => {
```

Comment:
Suggestion : Uniformiser l'écriture des fonctions dans ce fichier

Reply:
On pourrait se mettre d'accord sinon qu'on écrit toujours function() ? ou () => ?

Reply:
Ma suggestion concerne spécifiquement **ce** fichier où on doit utiliser `function` pour gérer le `this`

Reply:
Sans standard d'équipe je préfère ne pas prendre en compte ce changement, je marque comme resolve, je reste dispo pour en discuter ultérieurement